test/framework/build.gradle

```diff
@@ -88,6 +88,10 @@ test {
   systemProperty 'tests.gradle_index_compat_versions', BuildParams.bwcVersions.indexCompatible.join(',')
   systemProperty 'tests.gradle_wire_compat_versions', BuildParams.bwcVersions.wireCompatible.join(',')
   systemProperty 'tests.gradle_unreleased_versions', BuildParams.bwcVersions.unreleased.join(',')
+  
+  if (BuildParams.runtimeJavaVersion > JavaVersion.VERSION_17) {
```

Comment:
`>=`?

Reply:
Oh, in this case the change of security manager came into effect in JDK-18, so the condition is correct

Reply:
If I am being pedantic this should say `>= JavaVersion.VERSION_18` then?